pkg/interactive/validation.go

```diff
@@ -65,6 +68,9 @@ func IsCert(filepath interface{}) error {
 		if s == "" {
 			return nil
 		}
+		if s == "\"\"" {
+			return nil
+		}
```

Comment:
Why do you need this check? If the user enters "" as the actual string then that's wrong, we shouldn't let it through.

Reply:
@vkareh  so what should the user enter in order to delete the proxy value?

Reply:
It should be either `nil` or an empty string - if it's `nil`, no action is taken; if it's an empty string, the attribute is updated to an empty string

Reply:
in case I remove this check, I get:
**X Sorry, your reply was invalid: file '""' does not have a valid file extension**
? Additional trust bundle file path (optional): [? for help] 

There are three possible options of input from the user:
1) the user presses the '**enter**' button ---> rosa cli code returns either an existing value if exists, or double quotes ("") if no existing value. This is what happens regardless of the proxy feature.
2) the user enters double quotes ("') to remove an existing value --> rosa cli code returns "\"\"".  Again, regardless the proxy feature.
3) the user enters a valid value  ---> simple and straightforward case.

So the check above is for a case the user wants tp remove the current certificate by entering "" (like in OCM palyload), and we do not want to parse it.